### ISSUE TITLE
New patch R3nz dosent work

### ISSUE THREAD

**Issue by capychan** [hydy100/R3nzSkin#69]
riot launched 14.15 and r3nz dosen't work. Fix eta

**Comment by nebechodzz** [comment]
@hydy100 i downloaded from 14.14 and it says that the pattern is wrong

**Comment by nebechodzz** [comment]
![image](https://github.com/user-attachments/assets/4f19826c-7fc5-4919-9673-db8a0379b76c)

**Comment by querol999** [comment]
@hydy100 yeah it works on 14.15 perfect but what about new patch included skins ?

**Comment by Vendra35** [comment]
yeah we need new skins

**Comment by qingruxua** [comment]
ni hao da lao geng xing hou yun ding zhi yi yongbuliaole haigengxingma?
